src/tinyexpr.cpp

```diff
@@ -22,6 +22,15 @@
  * 3. This notice may not be removed or altered from any source distribution.
  */
 
+/* COMPILE TIME OPTIONS */
+
+/*
+ * Exponentiation associativity:
+ * For a^b^c = a^(b^c) do nothing
+ * For a^b^c = (a^b)^c comment the next line.
+ */
+#define TE_POW_FROM_RIGHT
```

Comment:
Please just remove the option entirely. This isn't a library anymore.

Reply:
Means just want only one version of pow? and I will squash changes..

Reply:
Done @faho :) Please check if that was expected sorry I hard pushed two times and amended changes with last commit